src/Razor/Microsoft.NET.Sdk.Razor/src/Microsoft.NET.Sdk.Razor.csproj

```diff
@@ -13,8 +13,6 @@
     <NoWarn>$(NoWarn);NU5100</NoWarn>
     <!-- It is intentional to not include build/netstandard2.0/Microsoft.NET.Sdk.Razor.targets -->
     <NoWarn>$(NoWarn);NU5129</NoWarn>
-    <!-- Ignore file copy warnings -->
-    <NoWarn>$(NoWarn);MSB3030;MSB4181</NoWarn>
```

Comment:
@pranavkm and @JunTaoLuo these warnings should be behind us, correct❔

Reply:
/ping @pranavkm @JunTaoLuo

Reply:
The first one should be fixed. I'm not sure what the second warning is for.:

> Emit MSB4181 when a task returns false (fails) without logging any errors

Reply:
We probably *do* want the second one to warn.